src/content/docs/cheerpj3/11-guides/Networking.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Networking
+description: Networking with CheerpJ
+---
+
+CheerpJ 3.0 supports different networking scenarios that are divided into two categories:
+
+- Same-origin HTTP/HTTPS requests using fetch seamlessly.
+- Generalized networking via Tailscale.
+
```

Comment:
IMO it's worth making it explicit that you must use the latter for anything that isn't HTTP(S), such as opening TCP or UDP sockets.